kythe/typescript/kythe.ts

```diff
@@ -0,0 +1,206 @@
+/**
```

Comment:
Add a @fileoverview at the top describing what this is.  Which I think is something like "exposes to TS the types and values from the kythe schema", or some such.

Reply:
Done.